liblouis/lou_translateString.c

```diff
@@ -1685,7 +1685,6 @@ insertBrailleIndicators(int finish, const TranslationTableHeader *table, int pos
 			if (!for_updatePositions(&indicRule->charsdots[0], 0, indicRule->dotslen, 0,
 						pos, input, output, posMapping, cursorPosition, cursorStatus))
 				return 0;
-			if (*cursorStatus == 2) checkWhat = checkNothing;
```

Comment:
How do you know this dead?

Reply:
I checked which functions are called from within doCompTrans.

Reply:
(... and insertBrailleIndicators is not one of them)

Reply:
OK